modular_skyrat/modules/emote_panel/code/emote_panel.dm

```diff
@@ -1023,4 +1023,4 @@
 /mob/living/proc/emote_mggaow()
 	set name = "> Mggaow"
 	set category = "Emotes+"
-	usr.emote("mggaow", intentional = TRUE)
\ No newline at end of file
+	usr.emote("mggaow", intentional = TRUE)
```

Comment:
you seem to have somehow removed the line ending here

Reply:
that diff means I added it, text editors usually do that if its missing automatically

Reply:
Checked out your branch and youre right